protected/models/Image.php

```diff
@@ -23,7 +23,8 @@
 class Image extends CActiveRecord
 {
     /** @const int  database id of the generic image (no_image.png) */
-    const GENERIC_IMAGE_ID = 0 ;
+    const GENERIC_IMAGE_ID  = 0 ;
+    const GENERIC_IMAGE_URL = "https://assets.gigadb-cdn.net/images/datasets/no_image.png";
```

Comment:
Probably worth adding a phpdoc to GENERIC_IMAGE_URL explaining it's for when users save the form without replacing the custom image they've removed